app/src/jira-client/delete-builds.test.ts

```diff
@@ -0,0 +1,31 @@
+import { deleteBuilds } from './delete-builds';
+import { Errors } from '../common/error-messages';
+
+describe('deleteBuilds suite', () => {
+    it('Should throw an error if no cloudId is passed', async () => {
+        (global as any).api = {
+            asApp: () => ({
+                __requestAtlassian: () => ({}),
+            })
+        };
+
+        try {
```

Comment:
/nit
Another way of doing it is https://jestjs.io/docs/expect#tothrowerror
One drawback in current pr is, when `await deleteBuilds(null)` doesn't throw, the test will PASS.

Reply:
Yeah I started implementing it that way initially but opted for this way so we can confirm the error being thrown in the one we expect. Doing this:

```
expect(() => {
    await deleteBuilds(null
  }).toThrow();
```

only allows us to confirm an error was thrown, not check which one.

Reply:
In the link above, it has some example may help
```
test('throws on octopus', () => {
  function drinkOctopus() {
    drinkFlavor('octopus');
  }

  // Test that the error message says "yuck" somewhere: these are equivalent
  expect(drinkOctopus).toThrow(/yuck/);
  expect(drinkOctopus).toThrow('yuck');

  // Test the exact error message
  expect(drinkOctopus).toThrow(/^yuck, octopus flavor$/);
  expect(drinkOctopus).toThrow(new Error('yuck, octopus flavor'));

  // Test that we get a DisgustingFlavorError
  expect(drinkOctopus).toThrow(DisgustingFlavorError);
});
```